src/core/cubic.c

```diff
@@ -398,6 +401,17 @@ CubicCongestionControlOnDataAcknowledged(
         // Slow Start
         //
 
+        const uint32_t MaxCwndGrowth =
+            (uint32_t)DatagramPayloadLength * QUIC_MAX_CONGESTION_WINDOW_INCREASE;
+
+        if (BytesAcked > MaxCwndGrowth) {
+            //
+            // Cap the CWND growth in slow start to prevent it from resulting in
+            // large bursts into the network.
+            //
+            BytesAcked = MaxCwndGrowth;
```

Comment:
This isn't quite right- we want to limit the SS increase, but not throw away the extra bytes in the case where we are proceeding into the CA path (since QUIC_MAX_CONGESTION_WINDOW_INCREASE doesn't apply in CA).

Reply:
You're talking about the case where `BytesAcked + CongestionWindow > SlowStartThreshold`? How would you recommend changing this?

Reply:
Yes. Something like
uint32_t NewCongestionWindow = Cubic->CongestionWindow + min(DatagramPayloadLength * QUIC_MAX_CONGESTION_WINDOW_INCREASE, BytesAcked);
BytesAcked -= NewCongestionWindow - Cubic->CongestionWindow;

then you'd need to change the "BytesAcked = Cubic->CongestionWindow - Cubic->SlowStartThreshold;" a few lines down to be a "+=".
CWND = NewCWND;

Reply:
I'm a bit confused how that logic would work. If cwnd is still < ssthresh, you want to set BytesAcked to `0`, right?

Reply:
Oh right. It would need to be more like

newcwnd = cwnd + min(MAX_CWND_INCREASE*MTU, BytesAcked);
if (newcmd >= ssthresh) {
    newcwnd = ssthresh;
    BytesAcked -= newcwnd - cwnd;
} else {
    BytesAcked = 0;
}
cwnd = newcwnd;

Your updated code looks like it works correctly, but consider the above if you think it might be cleaner.